Commit ID: 0776754c3f34963990173ba02ee5a765ab966e1a
Change ID: qrwltqroouvptlqntyozvptxmmynvuow
Author   : Lauri Tirkkonen <lauri@hacktheplanet.fi> (2025-11-01 14:01:43)
Committer: Lauri Tirkkonen <lauri@hacktheplanet.fi> (2025-11-01 14:18:22)

    test_rejected_input: skip on python>=3.12.12

    this test passes on 3.12.11, but fails on 3.12.12. since it mainly
    tests python stdlib behavior, and since upstream skips this test for
    >=3.13, we should work around by disabling the test for now.

diff --git a/bs4/tests/test_htmlparser.py b/bs4/tests/test_htmlparser.py
index 0ab0a411a9..f2cf6486ed 100644
--- a/bs4/tests/test_htmlparser.py
+++ b/bs4/tests/test_htmlparser.py
@@ -18,7 +18,7 @@
     default_builder = HTMLParserTreeBuilder
 
     # Fixed in https://github.com/python/cpython/issues/77057
-    @pytest.mark.skipif("sys.version_info >= (3, 13)")
+    @pytest.mark.skipif("sys.version_info >= (3, 12, 12)")
     def test_rejected_input(self):
         # Python's html.parser will occasionally reject markup,
         # especially when there is a problem with the initial DOCTYPE
